merlin/models/tf/models/ranking.py

```diff
@@ -202,44 +182,41 @@ def DeepFMModel(
     embedding_dim : int
         Dimension of the embeddings
     deep_block : Optional[Block]
-        The `Block` that learns high-ordeer feature interactions
-        Defaults to MLPBlock([64, 128])
+        The `Block` that learns high-order feature interactions.
+        On top of this Block, an MLPBlock([1]) is added to output
+        a 1-dim logit.
+        Defaults to MLPBlock([64])
+    input_block : Block, optional
+        The `Block` to use as the input layer for the FM and Deep components.
+        If None, a default `InputBlockV2` object
+        is instantiated, that creates the embedding tables for the categorical features
+        based on the schema, with the specified embedding_dim.
+        For a custom representation of input data you can instantiate
+        and provide an `InputBlockV2` instance.
     prediction_tasks: optional
         The prediction tasks to be used, by default this will be inferred from the Schema.
         Defaults to None
-    embedding_option_kwargs: Optional[dict]
-        Additional arguments to provide to `EmbeddingOptions` object
-        for embeddings tables setting.
-        Defaults to {}
     Returns
     -------
     Model
 
     """
-    input_block = InputBlock(
-        schema,
-        embedding_options=EmbeddingOptions(
-            embedding_dim_default=embedding_dim, **embedding_option_kwargs
-        ),
-        **kwargs,
-    )
 
-    pairwise_block = FMPairwiseInteraction().prepare(aggregation=StackFeatures(axis=-1))
-    deep_block = deep_block.prepare(aggregation=ConcatFeatures())
+    input_block = input_block or InputBlockV2(schema, dim=embedding_dim, aggregation=None, **kwargs)
```

Comment:
Perhaps it would be clearer to change the default `aggregation` in the InputBlock to be `None` and change the places where an aggregation is required to pass it?

Reply:
The only places where currently `InputBlockV2(...., aggregation=None)` are for `DeepFM` and `FMBlock`. So I it seems that is the exception and not the rule.